### DISCUSSION TITLE
Transmission web ui doesn't see completed torrents even though they are mounted in the container

### DISCUSSION THREAD

**Discussion by OvermindThe** [haugene/docker-transmission-openvpn#2447] [Q&A] [answered]
Hello,

Recently I had to move my docker to another machine and since then I've been having some problems.
This is the version that I'm using:
`haugene/transmission-openvpn   latest    49fc71236a76   10 days ago   259MB`

And this is my docker-compose which used to work (probably on the previous version of the image)
`version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/media/share/torrent:/data'
        environment:
            - OPENVPN_CONFIG=config
            - OPENVPN_PROVIDER=PROTONVPN
            - OPENVPN_USERNAME=xxxxxx
            - OPENVPN_PASSWORD=xxxxxxx
            - LOCAL_NETWORK=192.168.0.0/16
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn
        restart: unless-stopped`

However since I've started using this config on another machine my transmission web ui doesn't see completed torrents there.
I've checked in the container if the files get mounted correctly and they are - I can see the files there but they are not visible in the web ui.
They are in the /data/completed and /data/incomplete within the container
Also downloading files work - and they are getting persisted on my share, however when I restart docker image they are gone as well.

I've looked at the documentation http://haugene.github.io/docker-transmission-openvpn/ and the sample config looks the same.
Does the data folder need to have specific permissions for them to be correctly detected by transmission web ui?

Thank You for your help.

**Comment by creativecoder** [comment, marked as answer]
I think you're having the same problem I was having after the recent update. Downloading worked fine, but any config or previous downloads do not persist when restarting the container.

After some digging, I found that config has been extracted into a separate folder, so you need to mount an additional volume to persist the config. Try adding `/path/to/config:/config` to your list of volumes, where `/path/to/config` is a folder on your device that contains the `transmission-home` folder. That's what fixed it for me.

**Reply by dannylevenson** [reply]
Thanks for this! I had recently updated my config because I had to change my VPN, but that was the only thing that changed. After that, all of my completed torrents were missing. After adding the additional volume for /config, all the torrents are back. The interesting thing is that both /data and /config are now pointing to the same directory on the host file system.
